src/main/java/Duke.java

```diff
@@ -32,8 +32,8 @@
             "list: displays the complete list of tasks entered" + NEWLINE + "  Example:  list" + NEWLINE;
```

Comment:
Good use of constants here (comment failed to submit on last week tutorial so ignore this)